release.py

```diff
@@ -419,13 +431,30 @@ def tweak_patchlevel(
     print("done")
 
 
+def tweak_readme(tag: Tag, filename: str = "README.rst") -> None:
+    print(f"Updating {filename}...", end=" ")
+    readme = Path(filename)
+
+    # Update first line: "This is Python version 3.14.0 alpha 7"
+    # and update length of underline in second line to match.
+    lines = readme.read_text().splitlines()
```

Comment:
Should this check for empty lines at the beginning of the file and strip them, or are the first two lines guaranteed to be the header?

Reply:
I don't think we don't need to, the file starts immediately with the header and has done for a long time:

https://github.com/python/cpython/blob/main/README.rst?plain=1